packages/devtools-connect/package.json

```diff
@@ -76,7 +76,7 @@
     "gen-esm-wrapper": "^1.1.0",
     "mocha": "^8.4.0",
     "mongodb": "^5.8.1 || ^6.0.0",
-    "mongodb-log-writer": "^1.2.0",
+    "mongodb-log-writer": "^1.4.2",
```

Comment:
I don't even quite remember why, it must have been part of the SSDLC CodeQL work, but this was lying around in my local git checkout and there didn't really seem a reason not to just do this as a drive-by change, so I just commited it along with the actual change 🤷‍♀️